ingestion/src/metadata/ingestion/source/pipeline/fivetran/client.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, config: FivetranConnection):
         )
 
         client_config: ClientConfig = ClientConfig(
-            base_url=str(self.config.hostPort),
+            base_url=str(self.config.hostPort)[:-1],  # remove trailing slash
```

Comment:
@ayush-shah use the [clean_uri](https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/utils/helpers.py#L351) method instead